lib/index.js

```diff
@@ -349,6 +350,9 @@ function addValue({result, validate, key, values, label}) {
   if(Array.isArray(value)) {
     value = value.join(', ');
   }
+  if(value instanceof Date) {
+    value = value.getTime();
```

Comment:
My memory is that ISO 8601 dates are required in this specification.  Is that still the case?

Reply:
@mattcollier, it's both now, actually. We should be doing `getTime()` for params (pseudo headers) because they are unix timestamps now ... and `toUTCString()` for regular headers like the `Date` header. So -- @aljones15 -- you'll need to move things around a bit to accommodate for that.

Reply:
actually (created) and (expires) are unix time stamps:

https://tools.ietf.org/html/draft-cavage-http-signatures-11#section-2.1.4

> RECOMMENDED.  The `created` field expresses when the signature was
>    created.  The value MUST be a Unix timestamp integer value.

however you do have a point, if a client's middleware turned a common http header such as 
```
Date: Thu, 20 Jun 2019 16:29:11 GMT
```
into an actual date object that would result in the signing string containing
Date: 3423423423423 <-- not the actual unix time stamp
so it probably better to gate this behavior.

It does bring up an interesting issue because if Date is turned into a Date object and even if we do not turn it into a unix time stamp then the default toString method might not match the string from the client resulting in a signature mismatch.

Reply:
Addressed in https://github.com/digitalbazaar/http-signature-header/pull/13/commits/a0b7494eb2399263e4107128f79506cb9452dcf8

Provided no further comments I think we are good to go here.